blocktopmenu.php

```diff
@@ -1358,4 +1371,104 @@ public function renderList()
 
         return $helper->generateList($links, $fields_list);
     }
+
+    private function createCategoryNode(array $category, $addThumbnails = false)
+    {
+        if (!$category['active']) {
```

Comment:
In #40 I add legitimate need for displaying inactive categories.
Since it must be optional (and off by default), it means a new argument to the function prototype.
Would the maintainer agree on getting this done here now?